### PR TITLE
feat(auth): Allow command on user, password config attributes

### PR DESCRIPTION
**Pull request by lperdereau**

### CODE REVIEW

**Review by plaffitt — COMMENTED** [review]
LGTM. Maybe we could wait for #26 to be merged and write tests for `run_auth_commands` before merging, wdyt?

**Review by lcaflc — APPROVED** [review]
All Good.

**Comment by lperdereau** [comment]
> LGTM. Maybe we could wait for #26 to be merged and write tests for `run_auth_commands` before merging, wdyt?

Fail I merged, I will look to implement test ASAP

**Pull request merged**
